app/routes/inbox/trash.js

```diff
@@ -13,14 +14,15 @@ export default class InboxTrashRoute extends Route.extend(DataTableRouteMixin) {
     title: { refreshModel: true },
   };
 
-  mergeQueryOptions(params) {
+  model(params) {
     const query = {
       include: 'status,current-version',
```

Comment:
pagination is broken here because there is logic missing

Reply:
you're not passing the page size and number in params, so pagination is not working

Reply:
fixed, thanks for noticing it